node/follower_node.go

```diff
@@ -349,7 +349,11 @@ func (node *AlgorandFollowerNode) StartCatchup(catchpoint string) error {
 		node.log.Warnf("unable to create catchpoint catchup service : %v", err)
 		return err
 	}
-	node.catchpointCatchupService.Start(node.ctx)
+	err = node.catchpointCatchupService.Start(node.ctx)
+	if err != nil {
+		node.log.Warnf(err.Error())
+		return MakeStartCatchpointError(catchpoint)
```

Comment:
Include the error too?
```suggestion
		node.log.Warnf(err.Error())
		return MakeStartCatchpointError(catchpoint, err)
```

Reply:
Added in most recent commit.